chapters/Beacons.md

```diff
@@ -0,0 +1,587 @@
+* `beaconType` is one of "SingletonBeacon", "MapBeacon", or "SMTBeacon"
+* `serviceEndpoint` is a Bitcoin address represented as a URI following the [BIP21 scheme](https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki)
+
+How the Bitcoin address and the cryptographic material that controls it are generated is left to the implementation.
+
+**did:btc1** supports different ::Beacon Types::, with each type defining a set of algorithms for:
+
+1. How a ::BTC1 Beacon:: can be established and added as a service to a DID document.
+1. How ::BTC1 Beacon Announcements:: are broadcast within ::Beacon Signals::.
+1. How a resolver processes ::Beacon Signals::, identifying, verifying, and applying the authorized mutations to a DID document for a specific DID.
+
+This is an extensible mechanism, such that in the future new ::Beacon Types:: could be added.
+
+The current, active ::BTC1 Beacons:: of a DID document are specified in the document's `service` property. By updating the DID document, a DID controller can change the set of ::BTC1 Beacons:: they use to broadcast updates to their DID document over time. Resolution of a DID MUST process signals from all ::BTC1 Beacons:: identified in the latest DID document and apply them in the order determined by the version specified by the `didUpdatePayload`.
+
+All resolvers of **did:btc1** DIDs MUST support the ::Beacon Types:: defined in this specification.
+
+The ::BTC1 Beacon Announcement:: is a SHA256 hash that represents one of the following:
+
+* the hash of a single DID update payload associated with a **did:btc1** identifier;
+* the hash of a set of key-value pairs, where each key is a **did:btc1** identifier and the value is the hash of a DID update payload; or
+* the hash of an optimized ::sparse Merkle tree:: root, where each leaf node is deterministically selected by a **did:btc1** identifier and contains a hash associated with the **did:btc1** identifier.
+
+### Actors
+
+Actors in signaling are as follows:
+
+* DID controller - A party that controls one or more **did:btc1** identifiers participating in a ::BTC1 Beacon::.
+* Beacon Cohort - The set of unique cryptographic keys participating in a ::BTC1 Beacon:: that make up its n-of-n MuSig2 Bitcoin address.
+* Beacon Aggregator - The entity that coordinates the protocols of an aggregate ::BTC1 Beacon::, specifically the "Create Beacon Cohort" and "Announce Beacon Signal" protocols.
+* Beacon Participant - A member of a Beacon Cohort, typically a DID controller, that controls cryptographic keys required to partially authorize the broadcasting of a Beacon Signal to the Bitcoin blockchain.
+* Verifier - A party verifying a **did:btc1** identifier presentation.
+
+### Aggregation
+
+Three types of ::BTC1 Beacons:: are defined: SingletonBeacon, MapBeacon and SMTBeacon.  Two of them, MapBeacon and SMTBeacon, support aggregation, i.e. the act of  committing to multiple ::BTC1 Beacon Announcements:: in a single transaction.
+
```

Comment:
I would like to find a way to support referencing the terms Map Beacon etc here. Apprecaite that the type string is MapBeacon. Open to suggestions for how best to handle this